gabbi/httpclient.py

```diff
@@ -17,12 +17,32 @@
 import os
 import sys
 
-import httplib2
+import urllib3
 
 from gabbi import utils
 
 
-class VerboseHttp(httplib2.Http):
+class Http(urllib3.PoolManager):
+
+    def request(self, absolute_uri, method, body, headers, redirect):
+
+        retries = 1 if redirect else False
```

Comment:
I assume this is retries for a redirect? If so, you might want to be a bit more explicit like:

```
retries = False
if redirect:
    retries = urllib3.util.Retry(total=False, redirect=1)
```

Otherwise, that will do `Retry.from_int` which, if I am remembering correctly, sets the total number of retries. I don't know if you care about connection errors and other things being retried instead of redirects.